tests/ValueParsers/TimeParserTest.php

```diff
@@ -58,372 +55,278 @@ public function validInputProvider() {
 		$valid = array(
 			// Empty options tests
 			'+0000000000002013-07-16T00:00:00Z' => array(
-				TimeValue::newFromArray( array(
-					'time' => '+0000000000002013-07-16T00:00:00Z',
-					'timezone' => 0,
-					'before' => 0,
-					'after' => 0,
-					'precision' => TimeValue::PRECISION_DAY,
-					'calendarmodel' => TimeFormatter::CALENDAR_GREGORIAN
-				) ),
-				$emptyOpts,
+				new TimeValue(
+					'+0000000000002013-07-16T00:00:00Z',
```

Comment:
The parser tested here is not even capable of doing something with timezone, before and after other than setting it to 0. Repeating these three lines over and over again adds no information and makes the test cases very hard to read. Also note that this is only a base patch, #27 will remove the repeated zeros completely.